### DISCUSSION TITLE
Extend SqlException destructurer

### DISCUSSION THREAD

**Discussion by nskerl** [RehanSaeed/Serilog.Exceptions#23] [Q&A]
What is the best way to extend the [SqlException](https://github.com/RehanSaeed/Serilog.Exceptions/blob/master/Source/Serilog.Exceptions/Destructurers/SqlExceptionDestructurer.cs) destructurer? I'd like to include additional properties, like Procedure and ErrorCode. Is there a way to extend the existing destructurer or is it best to create a custom class. Thanks for any help.

**Comment by RehanSaeed** [comment]
You can create your own destructurer (See code for the current one for an example) and pass that to Serilog.Exceptions.
